src/saveload/afterload.cpp

```diff
@@ -3235,6 +3223,18 @@ bool AfterLoadGame()
 	/* Start the scripts. This MUST happen after everything else. */
```

Comment:
Maybe this comment needs some tiny amendment, since it's not the last thing anymore?